printer_mods/evandepol/Voron2.4-Cable-clamps/README.md

```diff
@@ -0,0 +1,36 @@
+# Cable clamps
+
+I used this set of clamps to route and secure the wiring in my Voron 2.4 350mm.
+
+There are three three types, each requiring an m3 bolt and t-nut in order to secure them to the inside perimeter of the upper (meaning closest to the deck) 2020 rail. The deck should rest on top of the clamps.
+
+![Example installation](Images/annotated.jpg)
+
+Each type of clamp has an internal channel for multiple small zipties, and the DIN rail clamps also provide secondary holes along the rails.
+
+openscad files are provided for your own tinkering needs.
+
+
+## Clamp types
+
+The first type only secures along the perimeter. I used this to secure high voltage wiring from the power inlet to the power supplies. Multi "tiers" of zip ties can be used to keep different types of wiring separated.
+
+![Clamp](Images/edge-clamp.png)
+
+The second type provides a horizontal bar that secures to the inside to the 2020 rail, and can be used to lash wiring to and keep it secure and organized. I used one of those to guide the high voltage wiring.
+
+![Horizontal bar](Images/cable-mount.png)
+
+The third type combines these two, and is used to route wiring along the perimeter, as well as guide nd secure wiring to various devices. I used several of these to provide wire tracks towards the Raspberry Pi, SKR boards, and SSR.
+
+![Combination](Images/combination.png)
+
```

Comment:
Both `Combination-Clamp-and-Ramp-350mm.stl` and `DIN-Rail-Cable-Mount-350mm.stl` have too steep of an overhang at the top of the mounting hole, please ensure overhangs are no steeper than 45 degrees.

<img width="372" alt="Screen Shot 2021-05-19 at 08 22 47" src="https://user-images.githubusercontent.com/69389593/118812351-025c7000-b87c-11eb-86c6-8567238c423c.png">

<img width="332" alt="Screen Shot 2021-05-19 at 08 23 24" src="https://user-images.githubusercontent.com/69389593/118812373-07212400-b87c-11eb-9320-ea1145b772ec.png">

Reply:
Working on this. The next commit on this branch should address this. Will post back when it's ready.

Reply:
Updated and simplified; this now uses bridging where the screw access hole is.